ui/StatusQ/src/StatusQ/Controls/StatusInput.qml

```diff
@@ -455,7 +455,7 @@ Item {
             onKeyPressed: {
                 root.keyPressed(event);
             }
-            onEditChanged: {
```

Comment:
No implications for this change?? It's a component used everywhere!! :)

Reply:
Yeah, that was a typo in the base component